jupytext/languages.py

```diff
@@ -24,7 +24,9 @@
                       '.ts': {'language': 'typescript', 'comment': '//'},
                       '.scala': {'language': 'scala', 'comment': '//'},
                       '.rs': {'language': 'rust', 'comment': '//'},
-                      '.robot': {'language': 'robotframework', 'comment': '#'}}
+                      '.robot': {'language': 'robotframework', 'comment': '#'},
+                      '.cs': {'language': 'csharp', 'comment': '//'},
+                      '.fs': {'language': 'fsharp', 'comment': '//'}}
```

Comment:
Two other file extensions for F#:

* `.fsx` - F# scripting file, can be loaded directly into the F# Interactive process
* `.fsi` - F# interface file (similar to OCAML interface files), used mostly in very large codebases to encapsulate code constructs

`.fsi` is probably not relevant to this project, but `.fsx` might be.